src/calibration_pipeline/DistributionUtils.jl

```diff
@@ -0,0 +1,24 @@
+function initial_parameter_ensemble(priors, N_ensemble; rng_seed = 10)
+    Random.seed!(rng_seed)
+    initial_ensemble = construct_initial_ensemble(priors, N_ensemble; rng_seed = rng_seed)
+    return initial_ensemble
+end
+
+#params::Dict{String, NamedTuple{(:mean, :constraint), Tuple{Float64, Constraint}}};
+function construct_priors(
+    params::Dict;
+    unconstrained_σ::Float64 = 1.0,
+    outdir_path::String = pwd(),
+    to_file::Bool = true,
+)
+    u_names = collect(keys(params))
+    u_mean_constraint = collect(values(params))
+    constraints = collect([v.constraint for v in u_mean_constraint])
+
+    # All vars are approximated as Gaussian in unconstrained space.
+    uncons_prior_mean = [c.constrained_to_unconstrained(μ_cons) for (μ_cons, c) in u_mean_constraint]
```

Comment:
https://clima.github.io/EnsembleKalmanProcesses.jl/dev/parameter_distributions/#constrained-gaussian

We always recommend the linked constructor. This does not require you to define prior distributions with parameters in the unconstrained space.  

you would provide, for example, `constrained_gaussian("name", mu_cons, sig_cons, 0, Inf)`

this may cut down some of this code.